ChangeLog

```diff
@@ -7,6 +7,7 @@
 ==== Major changes ====
  * Swapped InverseGamma shape/scale parameters: InverseGamma(k, lambda)
  * New Gaussian process regression classes
+ * Drop improvement factor attribute and accessors in EfficientGlobalOptimization
```

Comment:
I thought it would belong to "API changes" (not a major change)
the method name should be mentioned

Reply:
indeed, solved